### ISSUE TITLE
"Easter egg"? :D

### ISSUE THREAD

**Issue by maksimovic** [ForkIssues/Tracker#492]
I've just noticed the new year's greeting in the Fork's status bar at the top while checking out a new branch.

Well, happy new year to you Dan and Tanya 🎉 🎄 🎆

**Comment by DanPristupov** [comment]
Thank you!

And thank you very much for your always reasonable feedback and feature requests.

Happy New Year :).

**Comment by YesThatAllen** [comment]
Indeed, it was quite lovely to come back to work after the new year, and receive such a warm greeting from my favorite app!

**Comment by tandr** [comment]
Guys, you forgot to update the year - I just got a new year greeting, but the year was 2019 :)
